app/api/items/get_item.feature

```diff
@@ -137,7 +136,8 @@ Feature: Get item view information
         "can_grant_view": "enter",
         "can_view": "solution",
         "can_watch": "result",
-        "is_owner": true
+        "is_owner": true,
+        "has_can_request_help_to": false
```

Comment:
Why `has_can_request_help_to`  instead of `can_request_help` as suggested?
It looks more confusing to me.

Reply:
fixed